cms/settings/dev.py

```diff
@@ -71,7 +71,7 @@
 ENABLE_DJANGO_DEFENDER = False
 
 # Auth
-WAGTAIL_CORE_ADMIN_LOGIN_ENABLED = True
+SERVICE_AUTH_TOKEN = "wagtail-test-auth-token"  # noqa: S105 - Dummy token matching config in dis-authentication-stub
```

Comment:
Updated the value to clarify the intent and prevent confusion about it being a real token.